classes/sample_set.php

```diff
@@ -30,18 +30,53 @@ class sample_set {
     public $samples = [];
     public $starttime;
 
+    public $limit;
 
-    public function __construct($name, $starttime) {
+    public $filterrate = 1;
+    public $current = 0;
```

Comment:
A description here would be great. "Current..." what? Is this meant to be a counter / index / incremental id?